src/molecules/Copyable/Copyable.story.tsx

```diff
@@ -1,25 +1,38 @@
 import { storiesOf } from '@storybook/react';
 import React from 'react';
-
 import Typography from 'src/Typography';
+import styled from 'styled-components';
 import Copyable from './Copyable';
 
-function truncate(children: string) {
-  return [
-    children.substring(0, 6),
-    '…',
-    children.substring(children.length - 4),
-  ].join('');
-}
+const CopyableContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+`;
 
 storiesOf('Molecules', module).add('Copyable', () => (
   <>
-    <Typography>Before</Typography>
-    <Copyable
-      text="0x4bbeEB066D09B7AEd07bF39EE20460DFa261520"
-      truncate={truncate}
-    />
-    <Copyable text="foo.eth" />
-    <Typography>After</Typography>
+    <Typography>Copyable</Typography>
+    <CopyableContainer>
+      <Copyable
+        copyable={true}
+        text="0x4bbeEB066D09B7AEd07bF39EE20460DFa261520"
+      />
+      <Copyable
+        copyable={true}
+        truncate={true}
+        text="0x4bbeEB066D09B7AEd07bF39EE20460DFa261520"
+      />
+      <Typography>Non-Copyable</Typography>
+      <Copyable
+        copyable={false}
+        text="0x4bbeEB066D09B7AEd07bF39EE20460DFa261520"
+      />
+      <Copyable
+        copyable={false}
+        truncate={true}
+        text="0x4bbeEB066D09B7AEd07bF39EE20460DFa261520"
+      />
+    </CopyableContainer>
   </>
 ));
```

Comment:
I'm not seeing the copy icon in this story for some reason https://mycryptobuilds.com/ui/non-interactive-address-variant/?selectedKind=Molecules&selectedStory=Copyable&full=0&addons=1&stories=1&panelRight=0&addonPanel=%40storybook%2Faddon-a11y%2Fpanel